### ISSUE TITLE
Fringe does not match the background

### ISSUE THREAD

**Issue by andreyorst** [hlissner/emacs-doom-themes#605]
## What happened?

The following themes have wrong fringe color that doesn't match background of the window with solaire mode:

- `doom-1337`
- `doom-dark+`
- `doom-ephemeral`
- ~~`doom-material`~~
- `doom-moonlight`
- ~~`doom-palenight`~~
- `doom-sourcerer`
- `doom-zenburn`

Installed commit of doom-themes: 3ab4af2

**Comment by hlissner** [comment]
`doom-material` and `doom-palenight` can be removed from that list. Their fringe match.

As for the others, I noticed this but assumed that this was intentional.

cc @ccmywish (doom-1337), @ema2159 (doom-dark+), @karetsu (doom-ephemeral), @Brettm12345 (doom-moonlight), @fm0xb (doom-sourcerer), @jsoa (doom-zenburn)

Can you guys confirm? `fringe`'s and `default`'s `:background` don't match. Was this intentional?

**Comment by hlissner** [comment]
As a side-note, users can customize this with:

```elisp
;; use this after the theme has loaded
(custom-set-faces
 `(fringe ((t (:background nil)))))
```

**Comment by andreyorst** [comment]
> `doom-material` and `doom-palenight` can be removed from that list. Their fringe match.

on my machine they dont:

`doom-material`: ![image](https://user-images.githubusercontent.com/19470159/120098558-42a2d480-c13f-11eb-9c4d-c6a09ce4b8fb.png) - unreal buffer on the left, fringe, real buffer on the right.
`doom-palenight`: ![image](https://user-images.githubusercontent.com/19470159/120098583-6534ed80-c13f-11eb-9996-ac5bd402be43.png) - fringes are in between two real buffers.

Zoom in, it's a bit hard to see, but it's there.

**Comment by ccmywish** [comment]
1337-theme should be changed to **default bg**. Could you help me change it?

The theme was forked from another theme, so that  I didn't notice this, I'm sorry. Thanks for your carefulness.

**Comment by hlissner** [comment]
> on my machine they dont:
> Zoom in, it's a bit hard to see, but it's there.

@andreyorst Ah, that's the window border, not the fringe. They're `1px` wide, sit between every split, and don't (shouldn't) match the background in any of our themes.

**Comment by andreyorst** [comment]
> @andreyorst Ah, that's the window border 

No, it is not. The window border is visible on my screenshot as well, on the second one in particular. However I've updated today and the issue with these two themes gone. So I assume it was something else. Other themes still don't match.

**Comment by andreyorst** [comment]
@hlissner by the way, is it possible for minibuffer to automatically color fringes to the `solaire-minibuffer-face` color? Rightnow I have to either disable fringes in the minibuffer, or to color it to `solaire-default-face`. This may be because I'm using [mini-modeline](https://github.com/kiennq/emacs-mini-modeline) package though. IIRC solaire mode did that before the last update, but not for all themes.

![image](https://user-images.githubusercontent.com/19470159/120113464-3e000f80-c183-11eb-9637-bc07c251ac09.png)

**Comment by hlissner** [comment]
@andreyorst It can, but whether or not it's necessary depends on what version of Emacs you're using. Could you create an issue in the [solaire-mode](https://github.com/hlissner/emacs-solaire-mode/issues) repo for that?

**Comment by andreyorst** [comment]
sure

**Comment by ema2159** [comment]
Has it always been like this? Now I am not sure or maybe I didn't notice it. I've been using Ephemeral almost every day for more than a year and haven't noticed that until now. In my opinion, all of these should be changed. This issue is specially noticeable when using git gutter:
![image](https://user-images.githubusercontent.com/17773218/120862773-c6b6eb00-c589-11eb-950d-b74999311432.png)

**Comment by ema2159** [comment]
Closed via 4268ced
